azure-pipelines.yml

```diff
@@ -5,6 +5,12 @@ trigger:
     include:
       - master
       - maintenance/*
+variables:
+  # OpenBLAS_version should be updated
+  # to match numpy-wheels repo
+  OpenBLAS_version: 0.3.7.dev
+  TEST_GET_CONFIG: import numpy, ctypes; dll = ctypes.CDLL(numpy.core._multiarray_umath.__file__); get_config = dll.openblas_get_config; get_config.restype=ctypes.c_char_p; res = get_config(); print('OpenBLAS get_config returned', str(res)); assert b'OpenBLAS $(OpenBLAS_version)' in res
```

Comment:
I think it would be better to break the long line with `\` continuation characters. Leading space on the following line will be ignored, so you can indent it.

Reply:
Ok, I pushed that change in, but in a separate commit because I had a lot of formatting issues with that command. It gets used on multiple platforms / shells, and sometimes inside of other quotes, etc.

Reply:
Yeah, the failures are showing up already. I think I tried quite a few permutations and ultimately went with what worked which was a single-line approach.

Reply:
Shame, should work with yml. I wonder if putting the rhs in quotes would do it? Can make the experiment after this goes in.

Reply:
Alright, I removed the second revision commit. I think I played with quotes too with little success.

Reply:
Probably a script to execute the command would be better but a bit more complicated.